Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -522,12 +574,12 @@ extension FirefoxHomeViewController {
         return cell
     }
 
-    //should all be collectionview
     func configureTopSitesCell(_ cell: UICollectionViewCell, forIndexPath indexPath: IndexPath) -> UICollectionViewCell {
         let topSiteCell = cell as! ASHorizontalScrollCell
         topSiteCell.delegate = self.topSitesManager
-        topSiteCell.setNeedsLayout()
         topSiteCell.collectionView.reloadData()
+        topSiteCell.setNeedsLayout()
```

Comment:
Why did this need to be re-ordered?

Reply:
I think that might be an accident - I'll double check and revert!